clients/web/src/stylesheets/widgets/markdownWidget.styl

```diff
@@ -21,22 +24,39 @@
 
   .g-markdown-text
     height 100px
-    margin-bottom 4px
     padding 5px 9px
     background-color #fafafa
+    border 1px solid #ccc
     box-shadow inset 0 1px 2px rgba(0,0,0,0.075)
 
     &:focus
       background-color #fff
 
-    &[upload-enabled=true]
+    &[upload-enabled]
       border-bottom-left-radius 0
       border-bottom-right-radius 0
+      border-bottom 1px dashed #ccc
+
+    &.dragover
+      background-color $hoverBg
+      border-color $hoverBorderColor
+
+  .g-upload-footer
+    background-color #fafafa
+    color #777
+    border 1px solid #ccc
+    border-top none
+    padding 4px 10px
+    border-bottom-left-radius 3px
+    border-bottom-right-radius 3px
 
+    &.dragover
+      border-color $hoverBorderColor
 
   .g-markdown-preview
     min-height 80px
     padding 0 5px
+    overflow-x scroll
```

Comment:
I saw this in the modal dialog but not in the covalic app where it's not inside a modal, then tracked down why. This is from bootstrap:

```
.modal-body p {
    overflow: hidden;
    text-overflow: ellipsis;
}
```

I'll add some overrides.